data/xml/2022.sem.xml

```diff
@@ -0,0 +1,330 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2022.sem">
```

Comment:
This is the wrong venue code: it needs to be [starsem](https://aclanthology.org/venues/starsem/).

Reply:
I changed this.